solver.py

```diff
@@ -50,6 +56,13 @@ def create_board():
         
         sub_board.append(row)
     return sub_board
+# if the last function didn't this one definitely did
+
+def greeting():
+    while(1):
```

Comment:
this seems like it might break the program

Reply:
It shouldn't be an issue sir.